ios/Podfile.lock

```diff
@@ -906,9 +921,9 @@ SPEC CHECKSUMS:
   SDWebImageWebPCoder: 908b83b6adda48effe7667cd2b7f78c897e5111d
   Sentry: 1ed2d3f2973658bf6ab7ed43857c8e321a1625dd
   SocketRocket: fccef3f9c5cedea1353a9ef6ada904fde10d6608
-  Yoga: 99caf8d5ab45e9d637ee6e0174ec16fbbb01bcfc
+  Yoga: dc109b79db907f0f589fc423e991b09ec42d2295
   YogaKit: f782866e155069a2cca2517aafea43200b01fd5a
 
 PODFILE CHECKSUM: f8a1b9d32ad4447b452262ee2a33f0ffd98f6d4d
 
-COCOAPODS: 1.12.0
+COCOAPODS: 1.12.1
```

Comment:
is it important to upgrade all package versions?

Reply:
its because of react native minor update, I have the last xcode version installed and project dont builds without this update